### PR TITLE
[Chore] Modify some wrong or non-standard code

### PR DESCRIPTION
**Pull request by CalvinKirs**

### CODE REVIEW

File: seatunnel-core/seatunnel-core-base/src/main/java/org/apache/seatunnel/command/BaseTaskExecuteCommand.java [hunk 1]
```diff
@@ -85,7 +85,7 @@ protected final void close(List<? extends Plugin<E>>... plugins) {
             for (Plugin<E> plugin : pluginList) {
                 try (Plugin<?> closed = plugin) {
                     // ignore
-                } catch (Throwable e) {
+                } catch (Exception e) {
```
Comment (ruanwenjun) [inline seatunnel-core/seatunnel-core-base/src/main/java/org/apache/seatunnel/command/BaseTaskExecuteCommand.java:88]: As discussed at #1530, catch `Throwable` is better.

File: seatunnel-connectors/seatunnel-connectors-flink/seatunnel-connector-flink-doris/src/main/java/org/apache/seatunnel/flink/sink/DorisStreamLoad.java [hunk 2]
```diff
@@ -104,7 +104,7 @@ public void load(String value) {
     private LoadResponse loadBatch(String data) {
         String formatDate = new SimpleDateFormat("yyyyMMdd_HHmmss").format(new Date());
         String label = String.format("flink_sink_%s_%s", formatDate,
-                UUID.randomUUID().toString().replaceAll("-", ""));
+                UUID.randomUUID().toString().replace("-", ""));
```
Comment (ruanwenjun) [inline seatunnel-connectors/seatunnel-connectors-flink/seatunnel-connector-flink-doris/src/main/java/org/apache/seatunnel/flink/sink/DorisStreamLoad.java]: The UUID will generate String with 4 `-`, maybe use `replaceAll` is more reasonable?

**Review by ruanwenjun — COMMENTED** [review]
Good job, just add some comments.

File: seatunnel-core/seatunnel-core-base/src/main/java/org/apache/seatunnel/command/BaseTaskExecuteCommand.java (hunk 1 above)
Comment (CalvinKirs) [reply seatunnel-core/seatunnel-core-base/src/main/java/org/apache/seatunnel/command/BaseTaskExecuteCommand.java:88]: Throwable is the superclass of all errors and exceptions in Java. Error is the superclass of all errors, which are not meant to be caught by applications.

Comment (ruanwenjun) [reply seatunnel-core/seatunnel-core-base/src/main/java/org/apache/seatunnel/command/BaseTaskExecuteCommand.java:88]: Ok, make sense to me, we shouldn't catch `Error` in most of the time.

File: seatunnel-connectors/seatunnel-connectors-flink/seatunnel-connector-flink-doris/src/main/java/org/apache/seatunnel/flink/sink/DorisStreamLoad.java (hunk 2 above)
Comment (CalvinKirs) [reply seatunnel-connectors/seatunnel-connectors-flink/seatunnel-connector-flink-doris/src/main/java/org/apache/seatunnel/flink/sink/DorisStreamLoad.java]: String::replace does exactly the same thing as String::replaceAll without the performance drawback of the regex. just personal habit, , but there is no difference here.

Comment (CalvinKirs) [reply seatunnel-connectors/seatunnel-connectors-flink/seatunnel-connector-flink-doris/src/main/java/org/apache/seatunnel/flink/sink/DorisStreamLoad.java]: i restored it 🤣

Comment (ruanwenjun) [reply seatunnel-connectors/seatunnel-connectors-flink/seatunnel-connector-flink-doris/src/main/java/org/apache/seatunnel/flink/sink/DorisStreamLoad.java]: Ok, I tested and find they can return the same result.

**Review by ruanwenjun — APPROVED** [review]
LGTM

**Review by zhongjiajie — APPROVED** [review]

**Pull request merged**